lua/neogit/config.lua

```diff
@@ -153,13 +151,20 @@ M.values = {
   },
 }
 
-function M.ensure_integration(name)
-  if not M.values.integrations[name] then
-    vim.api.nvim_err_writeln(string.format("Neogit: `%s` integration is not enabled", name))
-    return false
+---@param name string
+---@return boolean
+function M.check_integration(name)
+  local logger = require("neogit.logger")
+  local enabled = M.values.integrations[name]
+
+  if enabled == nil or enabled == "auto" then
+    local success, _ = pcall(require, name)
+    logger.fmt_info("Found auto integration '%s = %s'", name, success)
+    return success
   end
 
-  return true
+  logger.fmt_info("Found explicit integration '%s' = %s", name, enabled)
```

Comment:
I'd like to start prefixing these with `[LOCATION]` to make reading the log easier. So, `[CONFIG]` in this case

Reply:
Like this e09dd40?

Reply:
Bang-on.